### DISCUSSION TITLE
关于web界面升级卡死的解决方法

### DISCUSSION THREAD

**Discussion by Boos4721** [Boos4721/openwrt#9] [Announcements]
- fw_setenv flag_last_success 0
- fw_setenv flag_boot_rootfs 0
- reboot
- 回到官方后 上传ubi至/tmp
- ubidetach -p /dev/mtd13
- ubiformat /dev/mtd13 -y -f /tmp/openwrt-ipq807x-generic-xiaomi_ax3600-squashfs-nand-factory.ubi
- nvram set flag_last_success=1
- nvram set flag_boot_rootfs=1
- nvram commit
- 然后重启即可

**Comment by WayneShao** [comment]
已经处理好，细节上是，如果另一个分区里不是官方，是QSDK，那命令用
```powershell
fw_setenv flag_last_success 1
fw_setenv flag_boot_rootfs 1
reboot
```
